lib/spack/spack/test/config.py

```diff
@@ -1203,28 +1203,6 @@ def test_internal_config_scope_cache_clearing():
     assert internal_scope.sections["config"] == data
 
 
-def test_system_config_path_is_overridable(working_env):
```

Comment:
Looks like these were deleted since they are tested in `test/paths.py`. I wonder if we want to keep them for an integration test for path resolution via config? Suggestion, not requirement if that seems like excessive test bloat.

Reply:
These tests are basically moved: they were interacting with private methods in `paths`, so they had to be rewritten, but the tests in the `paths` test module are meant to be direct analogs of these.